bridgeql/django/models.py

```diff
@@ -269,7 +269,9 @@ def _apply_opts(self):
                 self.qset = self.qset[:self.params.limit]
             elif isinstance(value, list):
                 # handle values case where property is passed in fields
-                if qset_opt == 'values' and self.query_has_properties():
+                if qset_opt == 'values' and stream:
+                    return self.yield_fields()
```

Comment:
Let's reform this to make a unified approach b/w return and non-return for each method, in this case this is being a special case here to return generator.
One more point regrading count query which is being evaluated after values operator, if count option is passed, stream will beof no use and should be set to False.